src/lsqecc/pipeline/lattice_surgery_compilation_pipeline.py

```diff
@@ -71,4 +72,7 @@ def compile_str(
         logical_computation, layout_types.SimplePreDistilledStates
     )
 
+    # TODO| when compilation stages are supported, remove the 'Circuit|' from the text
+    compilation_text += "\nCircuit| " + estimate_resources(lsc.composer.getSlices()).render_ascii()
```

Comment:
@Keelando , this line will conflict with #220, but it should be easy to fix. Ofc you can remove the comment and the `Circuit|` when merging 220. (probably want to keep the `\n` though)

Reply:
I'm still debating whether we should keep the compilation text, drop it or turn it into a "Log" stage. Because it's very useful to debug in plain text.